core/variables.tf

```
@@ -8,3 +8,13 @@ variable "secret_key_base" {
   description = "The secret key base for the application"
   type        = string
 }
+
+variable "rds_username" {
+  description = "RDS username"
+  type        = string
+}
```

for the sensitive variables, let's define them as sensitive to prevent the value leak to the log
```suggestion
variable "rds_username" {
  description = "RDS username"
  type        = string
  sensitive = true
}
```
https://developer.hashicorp.com/terraform/language/values/variables#suppressing-values-in-cli-output

---

same for other sensitive variables

---

It was updated in 83c83a3 ✅ Thank you!!!